.pylintrc

```diff
@@ -135,7 +132,7 @@ include-naming-hint=no
 inlinevar-rgx=[A-Za-z_][A-Za-z0-9_]*$
 
 # Regular expression matching correct method names
-method-rgx=(([a-z][a-z0-9_]{2,30})|(_[a-z0-9_]*))$
+method-rgx=(([a-z][a-z0-9_]{2,50})|(_[a-z0-9_]*))$
```

Comment:
Why need to increase from 30 to 50?
`{2,50}` means match at least two times but no more than 50 times, is it correct?
I see that [below](https://github.com/equinor/semeio/pull/518/files#diff-ba293004ae8f18975051e825bd8d5c70137a8b5d42d16e0f7c9354aa634c446cL156) we keep 30, so wondered the reason behind this change.

Reply:
There are some test function names that are longer than 30 characters. I think those test function names are legitimate and not something I want to disallow, so therefore I propose allowing up to 50 at least. One could think of having different requirements for test function names and production function names, but I think that just adds unnecessary complications to the CI.